sounds/views.py

```diff
@@ -271,13 +271,10 @@ def sound(request, username, sound_id):
                     sound.add_comment(request.user, comment_text)
                     sound.invalidate_template_caches()
                     try:
-                        if sound.user.profile.email_not_disabled("new_comment"):
-                            # Send the user an email to notify him of the new comment!
-                            logger.info("Notifying user %s of a new comment by %s" % (sound.user.username,
-                                                                                      request.user.username))
-                            send_mail_template(u'You have a new comment.', 'sounds/email_new_comment.txt',
-                                               {'sound': sound, 'user': request.user, 'comment': comment_text},
-                                               user_to=sound.user)
+                        # Send the user an email to notify him of the new comment!
+                        send_mail_template(u'You have a new comment.', 'sounds/email_new_comment.txt',
+                                           {'sound': sound, 'user': request.user, 'comment': comment_text},
+                                           user_to=sound.user, email_type_preference_check="new_comment")
                     except Exception as e:
```

Comment:
what's the difference between try/except here compared with checking the return value as is done in other places?

Reply:
There's no reason that I can think of. Also there is no reason for logging if this email sending fails so I just removed this code. e4aba1827ddc1691917bb941768a91766b3bf5aa